src/remove_duplicates.cpp

```diff
@@ -50,18 +53,15 @@ npe_arg(epsilon, double)
 npe_begin_code()
 
   assert_valid_tet_or_tri_mesh_23d(v, f);
-  // remove __copy
+  assert_nonzero_rows(v, "v");
+
   Eigen::MatrixXd v_copy = v.template cast<double>();
-  Eigen::MatrixXi f_copy = f.template cast<int>();
-  Eigen::MatrixXd nv;
-  Eigen::MatrixXi nf;
-  Eigen::VectorXi i;
-  igl::remove_duplicates(v_copy, f_copy, nv, nf, i, epsilon);
-  EigenDenseFloat nv_row_major = nv;
-  EigenDenseInt nf_row_major = nf.template cast<typename EigenDenseInt::Scalar>();
-  // FIXME: vector not allowing row major, but they should be essentially the same so i feel we can leave it as col major
-  Eigen::Matrix<typename EigenDenseInt::Scalar, Eigen::Dynamic, 1, Eigen::ColMajor> i_row_major = i.template cast<typename EigenDenseInt::Scalar>();
-  return std::make_tuple(npe::move(nv_row_major), npe::move(nf_row_major), npe::move(i_row_major));
+  Eigen::MatrixXd sv;
```

Comment:
@teseoch I could use guidance on how to refactor this function. remove_duplicates is no longer present in libigl. 

I see three options here: 
1. deprecate this function instead redirecting users to remove_duplicate_vertices 
2. keep this function and directly compute the currently missing "I" output (count of duplicates for each face)
3. keep this function but remove support for "I" parameter... 

Let me know how to proceed here, I'm unsure how users might be using this today.

Reply:
i think opt 2 is better